extensions/stackdriver/stackdriver.cc

```diff
@@ -515,9 +553,15 @@ void StackdriverRootContext::record() {
   ::Wasm::Common::RequestInfo request_info;
   ::Wasm::Common::populateHTTPRequestInfo(outbound, useHostHeaderFallback(),
                                           &request_info);
+  std::unordered_map<std::string,
```

Comment:
Can we add a typedef for this data structure?

Reply:
This should be in fewer places now. Please let me know if you think we should still typedef it.